server/package.json

```
@@ -87,7 +87,7 @@
     "yaml": "^2.2.1"
   },
   "dependencies": {
-    "@nomicfoundation/slang": "^0.10.1",
+    "@nomicfoundation/slang": "^0.13.1",
```

I suggest using an exact version here, to make sure we are actually using what we intend to use/bundle.

```suggestion
    "@nomicfoundation/slang": "0.13.1",
```

---

Makes sense; I was thinking about doing that as well, let me change this